### PR TITLE
:arrow_up::one::one: res_partner_skype Port

### PR DESCRIPTION
**Pull request by Ommo73**

### CODE REVIEW

File: res_partner_skype/__manifest__.py [hunk 1]
```diff
@@ -1,20 +1,52 @@
+# Copyright 2016 robyf70  <https://github.com/robyf70 >
+# Copyright 2019 Artem Rafailov <https://it-projects.info/team/Ommo73/>
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl.html).
 {
-    "name": "Skype field in partner form",
-    "version": "1.0.1",
+    "name": """Skype field in partner form""",
+    "summary": """Send private messages to specified recipients, regardless of who are in followers list.""",
```
Comment (KolushovAlexandr) [inline res_partner_skype/__manifest__.py]: What?

File: res_partner_skype/static/src/xml/base.xml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--Copyright 2014 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+    Copyright 2017 Ilmir Karamov <https://it-projects.info/team/ilmir-k>
+    Copyright 2019 Artem Rafailov <https://it-projects.info/team/Ommo73/>
+    License LGPL-3.0 (https://www.gnu.org/licenses/lgpl.html).-->
 <templates>
-<t t-name="FieldSkype" t-extend="FieldEmail">
+<t t-name="FieldSkype" t-extend="FieldReference">
```
Comment (KolushovAlexandr) [inline res_partner_skype/static/src/xml/base.xml]: Why here is reference field, but in js you extend mail one?

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]

File: res_partner_skype/__manifest__.py [hunk 3]
```diff
@@ -1,20 +1,51 @@
+# Copyright 2016 robyf70  <https://github.com/robyf70 >
+# Copyright 2019 Artem Rafailov <https://it-projects.info/team/Ommo73/>
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl.html).
 {
-    "name": "Skype field in partner form",
-    "version": "1.0.1",
+    "name": """Skype field in partner form""",
+    "summary": """Adds skype field to partner form. When you click on skype address, chat window is opened.""",
```
Comment (KolushovAlexandr) [inline res_partner_skype/__manifest__.py]: Adds skype field with a widget which opens skype chat window with the contact account when clicking on it

File: res_partner_skype/views.xml [hunk 4]
```diff
@@ -1,18 +1,22 @@
...
         <record id="view_partner_form_skype" model="ir.ui.view">
-             <field name="name">res.partner.form.skype</field>
+          <field name="name">res.partner.form.skype</field>
           <field name="model">res.partner</field>
           <field name="inherit_id" ref="base.view_partner_form"/>
           <field name="arch" type="xml">
                 <data>
                   <xpath expr="//field[@name='email']" position="after">
-                     <field name="skype" widget="skype"/>
+                        <field name="skype" widget="skype"/>
```
Comment (KolushovAlexandr) [inline res_partner_skype/views.xml]: it seems to be incorrect indent

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]

**Review by KolushovAlexandr — APPROVED** [review]

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

File: res_partner_skype/README.rst [hunk 5]
```diff
@@ -1,8 +1,51 @@
...
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
 
-Description: https://www.odoo.com/apps/modules/10.0/res_partner_skype/
+=============================
+ Skype field in partner form
+=============================
+
+Description: https://www.odoo.com/apps/modules/11.0/res_partner_skype/
```
Comment (yelizariev) [inline res_partner_skype/README.rst]: That link is already presented below

**Review by yelizariev — CHANGES_REQUESTED** [review]
Looks good :+1:

**Review by yelizariev — APPROVED** [review]

**Review by KolushovAlexandr — APPROVED** [review]

**Review by yelizariev — APPROVED** [review]

**Review by em230418 — CHANGES_REQUESTED** [review]
Cannot pass tests using odooup.
To reproduce:
```
odooup init test  # Odoo 11, no additional repo
cd test
bash
cd vendor
mkdir Ommo73 && cd Ommo73
git clone git@github.com:Ommo73/misc-addons.git -b 11.0-res_partner_skype_port
exit
make create && make build
docker-compose run dodoo -- init --new-database db --demo -m web,base,res_partner_skype
docker-compose run odoo -d db --test-enable --stop-after-init -i res_partner_skype
```

**Comment by yelizariev** [comment]
Superseded by #784

**Pull request closed without merging**